mmda/predictors/hf_predictors/bibentry_predictor/utils.py

```diff
@@ -3,7 +3,8 @@
 from mmda.types.annotation import SpanGroup
 from mmda.types.document import Document
 from mmda.types.span import Span
-from mmda.predictors.hf_predictors.bibentry_predictor.types import BibEntryPredictionWithSpan, BibEntryStructureSpanGroups
+from mmda.predictors.hf_predictors.bibentry_predictor.types import (BibEntryPredictionWithSpan,
+                                                                    BibEntryStructureSpanGroups)
```

Comment:
there are a ton of non-multiple-of-4 indentation added by this PR–are we ok with them?